boards/arm/rpi_pico/rpi_pico_w.dts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2023 Dave Rensberger - Beechwoods Software
```

Comment:
Combine first commit and last commit by squash.

Reply:
I assume that you wouldn't have any problem with my squashing _all_ of the comments into one for my next push?

Reply:
I combined all commits into a single summary commit now.